commandline/WSL/faq.md

```diff
@@ -12,25 +12,14 @@ ms.assetid: 129101ed-b88a-43c2-b6a2-cd2c4ff6fee1
 
 # Frequently Asked Questions
 
-> **Important note**  
-  This is the first release of this brand new technology and it is branded “beta” deliberately – it’s not yet complete! We know there are issues and incomplete features, you should expect some things to work and for many things to fail. But we appreciate you playing with this feature and helping us identify the issues we need to fix in order to deliver a great experience.
-
-
-### What is BASH?

```

Comment:
instead of removing "What is Bash" can we move this further down?  I think it's still nice to have a What is Bash section.  The start of the section could say "I've heard WSL referred to as Bash on Windows, What is Bash?"